numba/misc/codetype.py

```diff
@@ -0,0 +1,42 @@
+import types
+
+from numba.core import utils
+
+
+def copy_code_type(code: types.CodeType, **kwargs) -> types.CodeType:
```

Comment:
Is this essentially provided by stdlib `types.CodeType.replace()`? It's new in Python 3.8 but as that'll be the baseline version by the time 3.11 support is added should be ok to use?

Reply:
ok. i'll switch to it